hnn_core/network.py

```diff
@@ -1127,42 +1127,48 @@ def _instantiate_drives(self, tstop, n_trials=1):
                 self.external_drives[
                     drive['name']]['events'].append(event_times)
 
-    def add_tonic_bias(self, *, cell_type=None, amplitude=None,
-                       t0=None, tstop=None):
-        """Attach parameters of tonic biasing input for a given cell type.
+    def add_tonic_bias(self, *, cell_types_amplitudes, t0=0, tstop=None):
+        """Attach parameters of tonic biasing input for given cell types.
 
         Parameters
         ----------
-        cell_type : str
-            The cell type whose cells will get the tonic input.
-            Valid inputs are those in `net.cell_types`.
-        amplitude : float
-            The amplitude of the input.
+        cell_types_amplitudes : dic
```

Comment:
```suggestion
        cell_types_amplitudes : dict
```

Reply:
This is technically a breaking change ... can we just overload the parameter instead

Reply:
What do you mean by overload? Do we want to keep the existing functionality with the cell_type argument and single amplitude and have an additional argument for a dictionary specification?

Reply:
Keep the old functionality but throw a deprecation warning if `cell_type` is not `None`. For old functionality `amplitude` stays float. For new functionality `amplitude` is `dict`. That way you don't break old code that users have already written.

Reply:
I refactored the code to keep `add_tonic_bias ` old behavior and support the new one